src/util.h

```diff
@@ -175,4 +175,12 @@ static SECP256K1_INLINE void memczero(void *s, size_t len, int flag) {
     }
 }
 
+/** If flag is true, set *r equal to *a; otherwise leave it. Constant-time. */
```

Comment:
nit:  comment should also cover requirement to be defined, similar as in #754

Reply:
You're right :D fixed.